drivers/camera/Kconfig

```diff
@@ -0,0 +1,64 @@
+# Kconfig - Camera drivers
+
+#
+# Copyright (c) 2019 NXP Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+menuconfig CAMERA
+	bool "Camera Drivers"
+	help
+	  Enable camera drivers
+
+if CAMERA
+
+module = CAMERA
+module-str = camera
+
+config CAMERA_DISPLAY_DEV_NAME
```

Comment:
Should this be put under a boolean option, now it looks like a camera requires a display.

Reply:
this makes sense and I am trying add.